src/qt/coincontroldialog.cpp

```diff
@@ -430,11 +430,11 @@ void CoinControlDialog::viewItemChanged(QTreeWidgetItem* item, int column)
         else {
             coinControl->Select(outpt);
             int nRounds = vpwallets[0]->GetRealOutpointPrivateSendRounds(outpt);
-            if (coinControl->fUsePrivateSend && nRounds < privateSendClient.nPrivateSendRounds) {
+            if (coinControl->IsUsingPrivateSend() && nRounds < privateSendClient.nPrivateSendRounds) {
                 QMessageBox::warning(this, windowTitle(),
                     tr("Non-anonymized input selected. <b>PrivateSend will be disabled.</b><br><br>If you still want to use PrivateSend, please deselect all non-anonymized inputs first and then check the PrivateSend checkbox again."),
                     QMessageBox::Ok, QMessageBox::Ok);
-                coinControl->fUsePrivateSend = false;
+                coinControl->UsePrivateSend(false);
```

Comment:
I'd prefer to still call it a setter. Otherwise it implies it does more then that ("using privatesend in some way")

Reply:
As I described in the previous comment, these helpers are replacements for `fUsePrivateSend` which mirrors the state of the "PrivateSend" checkbox, so it felt ok-ish to name them this way. Would `UsePrivateSendCoins` be better? Or what setter name would you suggest?

Reply:
When I wrote this comment I assumed that there still was a boolean `fUsePrivateSend` inside coinControl and that `UsePrivateSend` would just set it to true. I'm ok now with the naming